packages/apps/transfer/src/services/transfer/coin-transfer.ts

```diff
@@ -0,0 +1,68 @@
+import { PactCommand } from '@kadena/client';
+import { sign } from '@kadena/cryptography-utils';
+
+import { onlyKey } from '../utils/utils';
+
+const NETWORK_ID = 'testnet04';
+const chainId = '1';
+export const API_HOST = `https://api.testnet.chainweb.com/chainweb/0.0/${NETWORK_ID}/chain/${chainId}/pact`;
+const gasLimit = 2300;
+const gasPrice = 0.00001;
+const ttl = 28800;
+
+export interface TransferResult {
+  requestKey?: string;
+  status?: string;
+}
+
+export async function makeTransferCreate(
+  fromAccount: string,
+  toAccount: string,
+  amount: string,
+  fromPrivateKey: string,
+): Promise<PactCommand> {
+  const pactCommand = new PactCommand();
+  pactCommand.code = `(coin.transfer-create "${fromAccount}" "${toAccount}" (read-keyset "ks") ${amount})`;
+
+  pactCommand
+    .addCap('coin.GAS', onlyKey(fromAccount))
+    .addCap<any>(
+      'coin.TRANSFER',
+      onlyKey(fromAccount),
+      fromAccount,
+      toAccount,
+      Number(amount),
+    )
+    .addData({ ks: { pred: 'keys-all', keys: [onlyKey(toAccount)] } })
```

Comment:
In the future we need to make this configurable. Could you create a task on the board for this?